app/controllers/company/opening.js

```diff
@@ -91,9 +91,30 @@ function getAllOpening(req, res, next) {
     res.status(200).json(fetchedInstance);
   });
 }
+
+/**
+ * get opening details by opening id
+ * @param  {Object}   req  Request Object
+ * @param  {Object}   res  Response Object
+ * @param  {Function} next Function to pass control to the next middleware
+ */
+
+function getOpeningDetails(req, res, next) {
+  openingLib.getOpeningDetails(req.user.openingId, req.body, function(
+    err,
+    fetchedInstance
+  ) {
+    if (err) {
+      res.status(500).json(err);
+      return;
+    }
+    res.status(200).json(fetchedInstance);
+  });
+}
 router.post('/addCredits', addCredits);
 router.post('/new', newOpening);
 router.put('/', updateOpeningDetails);
 router.delete('/', deleteOpening);
 router.get('/', getAllOpening);
+router.get('/', getOpeningDetails);
```

Comment:
This will not work since their are two routes with same name.

Reply:
changed the route name successfully